test/integration/fake_upstream.h

```diff
@@ -37,6 +37,7 @@
 
 #if defined(ENVOY_ENABLE_QUIC)
```

Comment:
can we use `#ifdef` or `#if defined(...` consistently? There's a mixture here. I see a lot more `#ifdef` in Envoy generally, so maybe change this one?